app/services/frameworks/importer.rb

```diff
@@ -35,5 +39,31 @@ def build_questions(framework, questions)
         questions.merge!(Frameworks::Question.new(filepath: question, questions: questions.dup).call)
       end
     end
+
+    def persist_framework(name, questions)
+      framework = Framework.new(name: name, version: version, framework_questions: questions.values)
+
+      if framework.valid?
```

Comment:
Perhaps change to `if framework.save` - that will return false if the record isn't valid so combines two lines in one